### DISCUSSION TITLE
AutoAttack Only pls!!!

### DISCUSSION THREAD

**Discussion by Acnexz** [ORelio/Minecraft-Console-Client#2261] [Q&A] [answered]
My sever is allowed to use Autoclick/AutoAttack but there must be no detection enttity, so i just need AutoAttack every 2 s. and eat when hungry No detection Entiity. 

***pls somebody help me.

**Comment by milutinke** [comment, marked as answer]
The AutoAttack can not work without Entity processing, it whole purpose is to attack entities, the protocol of the game does not allow for arbitrary click sending, we can not send clicks without a specified entity.

**Reply by Acnexz** [reply]
so saddd TuT
